data/fields/social_facility_for.json

```
@@ -1,5 +1,22 @@
 {
     "key": "social_facility:for",
     "type": "combo",
-    "label": "People Served"
+    "label": "People Served",
+    "strings": {
+        "options": {
+            "abused": "Abused",
+            "child": "Child",
+            "disabled": "Disabled",
+            "diseased": "Diseased",
+            "drug_addicted": "Drug Addicted",
+            "homeless": "Homeless",
+            "juvenile": "Juvenile",
+            "mental_health": "Mental Health",
+            "migrant": "Migrant",
+            "orphan": "Orphan",
+            "senior": "Senior",
+            "under_privileged": "Under Privileged",
```

(note that there's also a typo in the tag value)

```suggestion
            "underprivileged": "Under privileged people",
```